lib/dependency-manager-adapters/npm.js

```diff
@@ -21,8 +21,6 @@ module.exports = CoreObject.extend({
   configKey: 'npm',
   packageJSON: 'package.json',
   packageJSONBackupFileName: 'package.json.ember-try',
-  npmShrinkWrap: 'npm-shrinkwrap.json',
```

Comment:
@kategengler Not 100% sure, but can we remove backing up `npm-shrinkwrap.json` as well?

Reply:
Yeah I think so